yt/geometry/coordinates/coordinate_handler.py

```diff
@@ -184,98 +185,57 @@ def convert_to_spherical(self, coord):
     def convert_from_spherical(self, coord):
         pass
 
-    _data_projection = None
-
-    @property
+    @cached_property
     def data_projection(self):
-        if self._data_projection is not None:
-            return self._data_projection
-        dpj = {}
-        for ax in self.axis_order:
-            dpj[ax] = None
-        self._data_projection = dpj
-        return dpj
+        return {ax: None for ax in self.axis_order}
```

Comment:
So this is definitely not a user-facing issue, but from *my* perspective, I am wondering -- is it possible to invalidate the cached property at runtime?

Reply:
The result of a cached_property can be mutated transparently. Does this answer your question ?

Reply:
Oh I guess you were wondering if it was possible to reset the property to its initial state.
In which case the answer is: not to my knowledge

Reply:
... and a better answer yet, from the docs:
> The cached value can be cleared by deleting the attribute. This allows the cached_property method to run again.